Identity.API/Controllers/TokenController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Identity.BusinessLogic.DTOs;
+using Identity.BusinessLogic.Services.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Identity.API.Controllers
+{
+    [Route("api/token")]
+    [ApiController]
+    public class TokenController(ITokenService tokenService) : ControllerBase
+    {
+        private readonly ITokenService _tokenService = tokenService;
+
+        [HttpPost]
+        [Route("refresh")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> LoginAsync([FromBody] TokenDTO tokenDTO)
+        {
```

Comment:
you have a login method in another controller that handles authorization, rename this method